### DISCUSSION TITLE
Docusaurus Services Providers - Agencies and contractors

### DISCUSSION THREAD

**Discussion by slorber** [facebook/docusaurus#9281] [General]
This discussion is for agencies and contractors to introduce themselves to the community.

If you are providing, or looking to provide Docusaurus services to the community, now is the time to introduce yourself, and explain why someone should hire you to work on their Docusaurus site.

Please create one discussion item per provider.

Note: this discussion **only applies to paid work**. 
Providers in this list all expect to be paid for their services.
For free, benevolent open-source work, use [this other discussion](https://github.com/facebook/docusaurus/discussions/9283).

## Looking to hire a provider?

If you are interested to work with a provider, please contact them directly instead of answering here. 
You can eventually ask providers additional questions here.

You can also post your need on the dedicated issue: [Looking for a Docusaurus Service Provider?](https://github.com/facebook/docusaurus/discussions/9282)

Note this list of provider is "self-serve" and Docusaurus/Meta does not endorse them as "approved partners". If you hire a provider here, it remains your responsibility to select and vet them.

---

I suggest to use the following template, but feel free to include additional sections if you think it's useful.

## Suggested template:

```md
## Provider informations

- Detail here if you are you are an agency, freelancer, contractor or anything else.
- Include any detail can be useful, including your name, country, legal context you operate in...
- Mentioning a ballpark price estimate can also help readers to make a choice.
- Explain where/how to contact you.

## Provided services

What kind of Docusaurus service do you provide exactly?

Some examples:
- We sell monthly retainers to keep your Docusaurus site up-to-date
- We can setup i18n for your project
- We can audit your site and give you improvement ideas to reduce
- I can upgrade your site to Docusaurus v3
- I can handle implementation and maintenance of customizations through CSS and swizzling
- I can help make your docs more interactive with MDX and React
- I can implement custom Markdown syntax through MDX, Remark and Rehype
- I can help you host your Docusaurus site with the best performance settings
- I can reduce build time on your Docusaurus site

## Docusaurus experience

- How long have you been using Docusaurus?
- What challenging tasks have you worked on?
- Do you publish/maintain Docusaurus community packages?
- Do you have references working on open-source Docusaurus sites? 
- Any link you can provide to build trust?
```

**Comment by MaxSavonin** [comment]
We're a division of Keenethics, specializing in Docusaurus-related services, and currently in our second exciting year. Operating internationally, we adhere to all legal frameworks.

Our monthly basic maintenance plans start at $250. Reach out for custom quotes or more information at max.savonin@keenethics.com

**Services we offer:**

- Monthly maintenance plans for Docusaurus sites.
- Upgrades to Docusaurus v3.
- i18n setup for multi-language capabilities.
- Customizations via CSS and swizzling.
- MDX and React for enhanced interactivity.
- Advanced Markdown features.
- Build-time efficiency improvements.

**Regarding our Docusaurus expertise:**

As we navigate our second year, we've specialized in challenging tasks related to updating and maintaining documentation. We deal with localizing complex structures, including integrated interactive modules like MindMaps. Additionally, we improve visual representation and user experience in Docusaurus projects. Our in-house development focuses on AI integration in documentation, taking the search experience to the next level.
One of the projects we maintain [ShiftLeft Docs](https://docs.shiftleft.io/home).

**Comment by renatonagliati** [comment]
I'm a freelancer and offer services related to Docusaurus. I developed the tooltips and glossary plugin ([Remark auto glossary](https://github.com/renatonagliati/remark-auto-glossary)) and contributed on [learnprompting.com](https://learnprompting.org/docs/intro).

Reach out for custom quotes or more information at [renatonagliati@gmail.com](mailto:renatonagliati@gmail.com)

**Comment by dgarcia360** [comment]
We are [TechDocs Studio](https://techdocs.studio/), a specialized agency offering Docusaurus development and maintenance services. Based in Europe, we bring expertise in maintaining, customizing, and optimizing documentation for developer-focused products.

Our [maintenance plans](https://techdocs.studio/services/docusaurus-maintenance) start at $350/month, and [custom development services](https://techdocs.studio/services/custom-documentation-extensions) are available from $110/hour. 

For custom quotes, don’t hesitate to contact us at [david@techdocs.studio](david@techdocs.studio).

## Provided services

- Monthly maintenance plans for Docusaurus sites.  
- Theme customization for Docusaurus.  
- Custom extensions for Docusaurus.  
- Advanced search integrations.  
- Ask AI chatbot integration.  
- i18n configuration.  
- Documentation autogeneration from code.  
- Publication pipeline configuration.  

## Docusaurus experience

We've been working with Docs as Code setups for the past 6 years. Docusaurus has been our go-to documentation tool for the last 2 years. During this time, we've:

- Customized themes to align with brand needs.  
- Created custom extensions, such as an [AI chatbot for Docusaurus](https://docs.biel.ai/installation/docusaurus) and a [Feedback widget for Docusaurus](https://docs.pushfeedback.com/installation/docusaurus).
- Managed complex setups, orchestrating documentation publications across 15+ projects with a consistent look and feel.  

See [our portfolio](https://techdocs.studio/work) for examples of the projects we’ve worked on.
